package.json

```diff
@@ -105,7 +105,7 @@
     "classnames": "^2.2.0",
     "cozy-bar": "3.0.0-beta15",
     "cozy-client-js": "0.2.0",
-    "cozy-ui": "3.0.0-beta18",
+    "cozy-ui": "3.0.0-beta20",
```

Comment:
This means you'll need an new release of Cozy-UI, preferably before merging this PR, right?